models/fetchArticles.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection")
+
+function fetchArticles(sort_by = "created_at", order = "DESC") {
+    const validCols = ["comment_count", "created_at", "article_id", "title", "votes", "author", "topic"];
+    const validOrder = ["ASC", "DESC"];
+  
+    if (!validCols.includes(sort_by)) sort_by = "created_at";
+    if (!validOrder.includes(order)) order = "DESC";
```

Comment:
this task doesn't require taking different queries, but well done on creating the green lists for sort_by and order.
Throwing error messages when sort_by and order are invalid input, but it's task 11, so no need to change it for now.